src/js/msp/MSPHelper.js

```diff
@@ -1169,13 +1169,12 @@ MspHelper.prototype.process_data = function(dataHandler) {
                 // 0 = basic ledstrip available
                 // 1 = advanced ledstrip available
                 // Following byte is the current LED profile
-                let ledCount = (data.byteLength - 2) / 4;
+                let ledCount = (data.byteLength - 2) / 8;
 
                 for (let i = 0; i < ledCount; i++) {
+                    const mask = data.readU64();
```

Comment:
What happens if LED64 is not enabled? This works or crashes?

Reply:
It's not related to led count, both works. The 64 bit is the led configuration size.
This is the reason why I had to change it to 64 bit:
https://github.com/betaflight/betaflight/pull/12323#issue-1572957564

Reply:
But will work too with older firmware?

Reply:
Probably not, I should bind it to firmware version.

Reply:
This is easier to do with two 32 bits fields. One for led_low, and the other for led_high that will be zero for older version and only read for new version.

Reply:
I think this 64 bit function can be useful in the future.